packages/xo-server/src/xo-mixins/remotes.js

```diff
@@ -85,7 +85,16 @@ export default class {
   }
 
   async getAllRemotes() {
-    return (await this._remotes.get()).map(_ => obfuscateRemote(_))
+    const remotes = this._remotes.get().map(async remote => {
+      let infoDisk
+      if (remote.enabled) {
+        infoDisk = await this.getRemoteInfo(remote.id)
```

Comment:
What happens if this call is very long? (a few minutes)

Reply:
maybe i can add a timeout ?